Sources/FlowStack/FlowStack.swift

```diff
@@ -250,33 +261,45 @@ public struct FlowStack<Root: View, Overlay: View>: View {
         transaction.disablesAnimations = true
         return transaction
     }
-
+    @Environment(\.flowDismiss) var flowDismiss
     public var body: some View {
         ZStack {
             root()
                 .environment(\.flowDepth, 0)
+                .contentShape(Rectangle())
+                .accessibilityElement(children: .contain)
+                .accessibilityHidden(flowState.flowZIndex != 0)
+
 
             ForEach(pathToUse.wrappedValue.elements, id: \.self) { element in
                 if let destination = destination(for: element.value) {
 
                     skrim(for: element)
-
                     destination.content(element.value)
+                        .contentShape(Rectangle())
                         .frame(maxWidth: .infinity, maxHeight: .infinity)
                         .id(element.hashValue)
                         .transition(.flowTransition(with: element.context ?? .init()))
                         .environment(\.flowDepth, element.index + 1)
-                        .zIndex(Double(element.index) + 1)
+                        .zIndex(flowState.flowZIndex)
+                        .accessibilityElement(children: .contain)
+                        .accessibilityHidden(flowState.flowZIndex != Double(element.index + 1))
```

Comment:
❓ Could this just check the `element.index + 1 != pathToUse.wrappedValue.elements.count`?